server/src/service/IUserAvatarService.ts

```diff
@@ -0,0 +1,7 @@
+
+interface IUserAvatarService {
+
```

Comment:
**chore:** Remover linha em branco no inicio do arquivo e também no início do escopo da interface